### ISSUE TITLE
[Prova Level1] - [E-mail agendamento]

### ISSUE THREAD

**Issue by brunabtborges** [cwi-reset/edicao-04-level-1#350]
Bom dia a todos! Ainda não recebi o email com os horários disponíveis para a prova de sexta. Sabem informar se já foi enviado?
Além disso, eu vi em algum lugar, o site que eles utilizam para aplicar a prova, mas agora não lembro onde foi, se alguém puder enviar o link aqui, gostaria de dar uma olhada ;)

Desde já, agradeço!

**Comment by mayclau** [comment]
O site que a CWI usa para aplicar a prova é próprio, mas eles deixam a sugestão de alguns outros sites que usam uma metodologia parecida, ajuda bastante dar uma olhada e se familiarizar um pouco com a plataforma. 

https://www.codewars.com/
https://www.beecrowd.com.br/judge/en/login?origem=1

**Comment by brunabtborges** [comment]
> arecida, ajuda bastante dar uma olhada e se familiarizar um pouco com a p

Aaaah sim, era isso mesmo que eu estava procurando! rs muito obrigada!!

**Comment by brunabtborges** [comment]
Acabei de receber o email e agendar minha prova!! Boa sorte à todos!!! :)

**Comment by RebecaPR** [comment]
Boa tarde gente! 
eu vi que no e-mail da prova é necessário ter uma webcam mas que pode usar o celular como câmera externa, alguém sabe como faz isso?

**Comment by RebecaPR** [comment]
Ao agendar também, aparece o botão para agendar e quando se clica em cima ele passa para ✔ mas logo volta pra agendar novamente, como sei que foi confirmado o horário ?
![agendar](https://user-images.githubusercontent.com/110126661/191561490-bc9c604d-38f2-4332-a27b-5c7095d82324.PNG)

![image](https://user-images.githubusercontent.com/110126661/191561453-f685e92a-7017-4ec5-a7c6-f3048b6d2534.png)

**Comment by brunabtborges** [comment]
> Ao agendar também, aparece o botão para agendar e quando se clica em cima ele passa para ✔ mas logo volta pra agendar novamente, como sei que foi confirmado o horário ? ![agendar](https://user-images.githubusercontent.com/110126661/191561490-bc9c604d-38f2-4332-a27b-5c7095d82324.PNG)
> 
> ![image](https://user-images.githubusercontent.com/110126661/191561453-f685e92a-7017-4ec5-a7c6-f3048b6d2534.png)

também percebi isso! Eu espero que esteja agendada mesmo, mas já poderíamos reportar esse erro aí pra melhor a usabilidade! rsrs pelo menos mandar uma mensagem de confirmação.

**Comment by NicoledeRossi** [comment]
Também não recebi mensagem de confirmação. QA deixou passar essa kkkkkk

**Comment by Felipe-Koch** [comment]
Eu não recebi o e-mail tb! Mais alguém?

**Comment by CamilaGoulart** [comment]
Talvez não tenha uma confirmação de agendamento, pois você pode mudar esse agendamento, até amanhã meio dia, como diz no e-mail, mas de qualquer forma, o meu quando atualiza, fica apagado a parte de "agendar" fica em preto e branco, ( e fica aparecendo o horário que marquei) se eu quiser mudar o horário, ai consigo alterar a hora e o "agendar" volta a ficar amarelo.

**Comment by brunabtborges** [comment]
Boa Camila!! Entrei de novo no site e realmente o botão ficou cinza ;)

**Comment by Askalor** [comment]
> Eu não recebi o e-mail tb! Mais alguém?

Também não recebi. Normalmente os emails do Reset levam alguns dias para chegar aqui. Os e-mails sobre a live (com o formulário) nunca chegam no dia que o Guilherme diz manda-los. Nem mesmo na caixa de spam aparece

**Comment by brunabtborges** [comment]
> Eu não recebi o e-mail tb! Mais alguém?

https://reset.cwi.com.br/candidato

**Comment by brunabtborges** [comment]
> > Eu não recebi o e-mail tb! Mais alguém?
> 
> Também não recebi. Normalmente os emails do Reset levam alguns dias para chegar aqui. Os e-mails sobre a live (com o formulário) nunca chegam no dia que o Guilherme diz manda-los. Nem mesmo na caixa de spam aparece

https://reset.cwi.com.br/candidato

**Comment by Allemaun** [comment]
> Boa tarde gente! eu vi que no e-mail da prova é necessário ter uma webcam mas que pode usar o celular como câmera externa, alguém sabe como faz isso?

Boa tarde @RebecaPR 

Usa o DoidCam é um app bem fácil de entender o uso, procura no Youtube tem alguns vídeos bem curtos e diretos.
Mas já te adianto que é mais fácil o uso caso teu Note o Desktop tenham conexão via wifi, mas é possível fazer conexão via cabo USB.

**Comment by Allemaun** [comment]
> Ao agendar também, aparece o botão para agendar e quando se clica em cima ele passa para ✔ mas logo volta pra agendar novamente, como sei que foi confirmado o horário ? ![agendar](https://user-images.githubusercontent.com/110126661/191561490-bc9c604d-38f2-4332-a27b-5c7095d82324.PNG)
> 
> ![image](https://user-images.githubusercontent.com/110126661/191561453-f685e92a-7017-4ec5-a7c6-f3048b6d2534.png)

@RebecaPR 

Se vc sair da página e entrar novamente ao abrir o agendamento vai estar selecionado o horário selecionado, serve como confirmação

**Comment by cristianodc** [comment]
Como sei que o horario esta reservado?? a gente recebe um email de confirmação ??

**Comment by RebecaPR** [comment]
@cristianodc é isso que eu questionei e o pessoal estava me explicando... a gente não recebe e-mail não... mas como a colega explicou, como podemos mudar nosso horário até 12h do dia 22, acredito que eles deixaram em aberto...

**Comment by guilhermewmuller** [comment]
A confirmação do agendamento será enviada por e-mail amanhã, juntamente com o link de acesso a prova. Apenas o último registro de agendamento será considerado!

**Comment by guilhermewmuller** [comment]
> > Eu não recebi o e-mail tb! Mais alguém?
> 
> Também não recebi. Normalmente os emails do Reset levam alguns dias para chegar aqui. Os e-mails sobre a live (com o formulário) nunca chegam no dia que o Guilherme diz manda-los. Nem mesmo na caixa de spam aparece

Sim, os e-mails podem demorar para chegar! Lembrando que os links disponibilizamos sempre aqui no GIT: https://github.com/cwi-reset/edicao-04-level-1#calend%C3%A1rio-de-lives

**Comment by Askalor** [comment]
> > > Eu não recebi o e-mail tb! Mais alguém?
> > 
> > 
> > Também não recebi. Normalmente os emails do Reset levam alguns dias para chegar aqui. Os e-mails sobre a live (com o formulário) nunca chegam no dia que o Guilherme diz manda-los. Nem mesmo na caixa de spam aparece
> 
> Os links disponibilizamos sempre aqui no GIT: https://github.com/cwi-reset/edicao-04-level-1#calend%C3%A1rio-de-lives

Obrigado pela resposta @guilhermewmuller. Quanto as lives eu realmente sempre olhei pelo GIT e não pelo meio email. Já os formulários eu já preenchia assim que enviados nas lives também. Então nesse quesito eu não tenho preocupação.
O que me preocupa é por exemplo o que já foi dito aqui na issue em respeito a precisarmos de uma câmera para a prova e também o link para fazer o agendamento. Se o email não chegar até amanhã (e levando em conta que eles levam em média 2~3 dias pra chegarem para mim) eu perderia o agendamento, além das informações extras que contem no email e não estão no FAQ/site de agendamento/GIT. A outra preocupação é em caso acabar selecionado e precisar agendar a entrevista. Suponho que também será pela page de candidato, mas poderei perder informações extras contidas no email.

**Comment by guilhermewmuller** [comment]
@Askalor caso não receba os e-mails relacionados a prova dentro dos prazos que estamos divulgando, peço que sinalize no e-mail reset@cwi.com.br com um telefone de contato, para que possamos lhe passar as informações por outro meio.

**Comment by MeCrepaldi** [comment]
Ainda não recebi o email referente a prova, o que houve?

**Comment by guilhermewmuller** [comment]
@MeCrepaldi caso ainda não tenha recebido, verifique sua caixa de Spam. Se ainda assim não localizar o e-mail, por gentileza envie um e-mail para reset@cwi.com.br.
